index.html

```diff
@@ -240,21 +255,25 @@ <h2 id="digest_error" class="error" style="display:none;">
     <!--##########################-->
     <!--##  Step 4: Challenges  ##-->
     <!--##########################-->
-    <h2>Step 4: Verify Ownership<span id="step4_pending"> (waiting...)</span></h2>
+    <h2>Step 4: Verify Ownership<span id="step4_pending"> (waiting…)</span></h2>
     <div id="step4" style="display:none;">
-        <div>
-            Let's Encrypt requires you prove you own the domains you have in your CSR. You
+        <p>
+            Let’s Encrypt requires you prove you own the domains you have in your CSR. You
             can do this by serving a specific file at a specific url under your domains.
             Below are the files you need to serve along with some copy-and-paste commands
             you can run on your website to start serving the file. Once you are serving
-            the file on your website, click "I'm now running this on...". After that, you
-            need to tell Let's Encrypt to check the above files to verify ownership of your
+            the file on your website, click “I’m now running this on…”. After that, you
+            need to tell Let’s Encrypt to check the above files to verify ownership of your
             domains. This request needs to be signed with your account private key. Below
             are the verification requests that you will need to sign. The commands to do
             this are generated below so you can copy-and-paste them into your terminal.
             <i>Be sure to change the account private key location so it points to your real
             private key.</i>
-        </div>
+        </p>
+        <h4>Renewal Note</h4>
+        <p>
+            If you’re renewing a certificate, previously used the file-based method, and did not delete the verification file from your server, you should not need to regenerate the signature but can simply verify using the existing file.
```

Comment:
These instructions could be more clear.  I'm not sure what they actually mean.  Should I not use the `Run this signature command in your terminal:` form?